.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,36 @@
-- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+<!--
+Thank you for sending the PR! 
+Please fill the applicable details below
+Happy contributing!
+-->
 
-- **Why was this change needed?** (You can also link to an open issue here)
+### What kind of change does this PR introduce? 
+<!-- Mark All The Applicable Boxes -->
 
-- **Other information**:
+- [ ] Bug
+- [ ] Feature
+- [ ] Docs
+- [ ] Other(s)
+
+
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+### Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+### Why was this change needed?
+
+<!-- Explain all the changes in the PR and tell what is the use of this change. -->
+
+
+### Screenshots
```

Comment:
This text could be moved into "Other information"

Reply:
No, I think Screenshots should be a different heading only so that's easy
What to other think @scopsy

Reply:
Screenshots are only needed for UI changes and because of that I think it belongs in "Other information"

Reply:
Okay!
Should I do the changes then @davidsoberg

Reply:
@MrKrishnaAgarwal please follow @davidsoderberg suggestion.